packages/dataviews-client/src/resolve-dataviews-generators.ts

```diff
@@ -46,6 +46,32 @@ type DataviewsGeneratorConfigsParams = {
 
 type DataviewsGeneratorResource = Record<string, Resource>
 
+const getUTCDate = (timestamp: number) => {
+  const date = new Date(timestamp)
+  return new Date(
+    Date.UTC(
+      date.getFullYear(),
+      date.getMonth(),
+      date.getDate(),
+      date.getHours(),
+      date.getMinutes()
+    )
+  )
+}
+
+const getDatasetsExtent = (datasets: Dataset[] | undefined) => {
```

Comment:
Nice 👍
Would be actually cool to have that in a selector and send to Timebar's `absoluteStart|End` instead of using hardcoded `DEFAULT_WORKSPACE`